spec/models/concerns/courses/edit_options/qualifications_concern_spec.rb

```diff
@@ -18,7 +18,7 @@
     let(:level_value) { "primary" }
 
     it "returns only QTS options for users to choose between" do
-      expect(example_model.qualification_options).to eq(%w[qts pgce_with_qts pgde_with_qts])
+      expect(example_model.qualification_options).to match_array(%w[qts pgce_with_qts pgde_with_qts])
```

Comment:
The order of laziness, thats just too lazy.

Reply:
hah, yeah I didn't wanna have the tests tied to the ordering if possible :man_shrugging: then again we're saying the ordering is important in the backend so i guess they should be

Reply:
Its both really subtle and really obscured.
So there is a real need for tests here.